README.md

```diff
@@ -31,6 +31,23 @@ Available options:
 
 ## Usage
 
+`can-ssr` can be used either as Express middleware or programatically.
+
+### Express Middleware
+
+Use the provided middleware to add server-side rendering to an existing Express server:
+
+```js
+var ssr = require(‘can-ssr/lib/middleware’),
```

Comment:
If we're going to document this then it's part of the API.  In which case I think it would be nicer to do `require("can-ssr/middleware")`.  So we could have a `middleware.js` in the root folder that just does:

```js
module.exports = require("./lib/middleware");
```